network-api/networkapi/news/factory.py

```diff
@@ -32,34 +32,15 @@ class Meta:
         exclude = ("headline_sentence",)
 
     class Params:
-        unpublished = Trait(
```

Comment:
Similarly, these details seem a bit unnecessary to me. I'm deleting them to avoid complication.

Reply:
Will add them back!